tool/doc.dart

```diff
@@ -108,6 +127,12 @@ Future<void> generateDocs(String dir) async {
   new OptionsSample(rules).generate(outDir);
 }
 
+String getBadges(String rule) => isPedantic(rule)
+    ? '<a href="https://github.com/dart-lang/pedantic/#enabled-lints"><img alt="pedantic" src="pedantic-1.4.svg"></a>'
```

Comment:
I thought about that.  The trouble is that, in theory, some versions may remove some lints so it could get even more complicated.  Another alternative is to dispense with the version entirely and just do something like:

![image](https://user-images.githubusercontent.com/67586/49590437-c2f66f00-f920-11e8-9de7-87757051956d.png)

Reply:
@davidmorgan: thoughts on whether we should badge version number?

Reply:
Dropping the version number seems a good thing to start with. You'd anyway need to signal ranges to convey the full information :)

Reply:
I agree. The version number doesn't mean much at this point. It would be good to eventually have a range of versions in which it is enabled so that users can see which versions of pedantic to specify, but let's not show anything until we get to that point.

Reply:
OK.  We're all on the same page.  I've updated the badge to be version-less.  See the original comment (updated) to see in situ, but here's there idea:

![image](https://user-images.githubusercontent.com/67586/49604228-3e1c4d00-f942-11e8-9283-359136e08768.png)


(FWIW: this will work more consistently with other badges I have in mind -- e.g., for `flutter` -- so this is great!)